If the config option `projects` is empty, do not omit any entries. (#8)
minddistrict/gtimelog2tick: CHANGES.rst

- Fetch all projects, not just the first 100.

> - If the config option ``projects`` is empty, do not omit any entries.

Reviewer: ```suggestion
- If the config option ``projects`` is empty, upload all possible entries.
```
Too much negation.
